exodus_lambda/functions/origin_request.py

```diff
@@ -67,6 +66,14 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         uri = self.resolve_aliases(request["uri"])
 
+        self.logger.info(
```

Comment:
Anyone want to share some thoughts on whether INFO or DEBUG is appropriate here?

We should probably have some rough guidelines to tell us when a log message might occur too often or be too verbose for INFO level.  Maybe a log message which occurs once per request (like these) is fine for INFO, while a log message which occurs many times per request (e.g. in a loop) would be better as DEBUG.

Reply:
I like that guideline, myself. These do make enough sense to me as INFO messages.